xbit/routes/controls/user.js

```diff
@@ -0,0 +1,110 @@
+
```

Comment:
for backend code, it's not a good idea to write
exports.foo = ...
exports.bar = ...

Instead, we should write
var module = {
foo: ...,
bar: ..
}

or 
var module = {};
module.foo = xxx
module.bar = yyy

then:
exports = module

The exports have dependencies, sometimes we'll get into a dependency-hell ...